plugins/topology.yaml

```diff
@@ -0,0 +1,84 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: topology
+spec:
+  version: v0.0.1
+  shortDescription: Provides insight into the topology of a Kubernetes cluster
+  homepage: https://github.com/bmcstdio/kubectl-topology
+  description: |
+    Provides insight into the topology of a Kubernetes cluster (i.e. into the
+    distribution of nodes and pods per region and zone).
+
+    # List all nodes in the cluster and the zones and regions they belong to.
+    $ kubectl topology node
+    NAME     REGION         ZONE
+    node-1   europe-west1   europe-west1-a
+    node-2   europe-west1   europe-west1-a
+    node-3   europe-west1   europe-west1-b
+    node-4   europe-west1   europe-west1-c
+
+    # List nodes in the 'europe-west1-a' zone only.
+    $ kubectl topology node --zone europe-west1-a
```

Comment:
I think 1 example is more than enough. 
You can probably remove the rest And make the last paragraph more prominent.

Reply:
Done!